docs/analysis/interpretation/crossSections.md

```diff
@@ -0,0 +1,20 @@
+# Cross section calculation
+
+!!! Warning
+    This page is under construction
+
+Cross sections can be calculated for MC samples. The starting page is [GenXSecAnalyzer](https://twiki.cern.ch/twiki/bin/viewauth/CMS/HowToGenXSecAnalyzer). Note that this link is currently restricted to CMS collaborators.
+
+To account for the different running conditions in Run 1 vs Run 2, click the appropriate tab below for Run 1 vs Run 2 data.
+
+=== "Run 1 Data"
+
+    * This page is under construction
+
+=== "Run 2 Data"
+
+    * To compute the cross-section using the GenXSecAnalyzer, you can use a docker container. You can find a list of Docker container images available for CMS open data in the [guide page for CMS open data containers](http://opendata.cern.ch/docs/cms-guide-docker). A tutorial on working with docker is at the [CMS open data containers](https://cms-opendata-workshop.github.io/workshop2022-lesson-docker/).
+    * To calculate a cross-section, follow the directions in the section "Running the GenXSecAnalyzer on an existing MC sample" in [GenXSecAnalyzer](https://twiki.cern.ch/twiki/bin/viewauth/CMS/HowToGenXSecAnalyzer).
```

Comment:
@jieunyoo This twiki page is CMS internal and external users can't get to it. You will need to add the relevant commands explicitly on this page. Thanks!